jdaviz/app.py

```diff
@@ -311,7 +318,10 @@ def get_data_from_viewer(self, viewer_reference, data_label=None, cls=None,
             corresponding viewer data labels.
         """
         viewer = self.get_viewer(viewer_reference)
-        cls = cls or viewer.default_class
+        cls = viewer.default_class if cls == 'default' else cls
+        if not isclass(cls):
```

Comment:
This is not correct: `cls` can be a class object, 'default' or `None` (and in fact `None` is necessary case, hence the failing tests.)